### PR TITLE
ETC Course: 2. The Cypherpunks

### PR DESCRIPTION
**Pull request by DonaldMcIntyre**
This is class 2 of the ETC Course that will be published on 2022-12-29. CN translation coming soon.

### CODE REVIEW

**Comment by w1g0** [comment]
@DonaldMcIntyre, maybe keep it in draft until the CN version is added?

**Review by TheCrowbill — APPROVED** [review]

**Comment by DonaldMcIntyre** [comment]
> @DonaldMcIntyre, maybe keep it in draft until the CN version is added?

Yes, soory I forgot to move it to draft.

**Comment by bobsummerwill** [comment]
Cypherpunks group formed 1992 as per NYT, not 80s.  Interesting article here:

https://www.nytimes.com/2018/12/21/obituaries/timothy-c-may-dead.html?smid=nytcore-ios-share&referringSource=articleShare

**Comment by bobsummerwill** [comment]
https://www.coindesk.com/markets/2018/10/19/enough-with-the-ico-me-so-horny-get-rich-quick-lambo-crypto/ I had not seen before.

**Comment by DonaldMcIntyre** [comment]
> Cypherpunks group formed 1992 as per NYT, not 80s.  Interesting article here:
> 
> https://www.nytimes.com/2018/12/21/obituaries/timothy-c-may-dead.html?smid=nytcore-ios-share&referringSource=articleShare 

Tim May told me he started it in 1988.

**Comment by DonaldMcIntyre** [comment]
> https://www.coindesk.com/markets/2018/10/19/enough-with-the-ico-me-so-horny-get-rich-quick-lambo-crypto/ I had not seen before.

Yes. This was his style. He wrote that text as a disorganized stream of thoughts, then got tired and sent the text to Coindesk and told them to do what they wanted with it. He passed away two months after. Very sad.

**Comment by DonaldMcIntyre** [comment]
@bobsummerwill, I see what you mean. Yes, the formal naming and activities started in the early 90s so I changed the wording to add "early 90s". I also added text to clarify that "the individuals who would later become the Cypherpunks" in the 1980s. So, this is more accurate. Thank you!

File: content/blog/2022-12-29-ethereum-classic-course-2-the-cypherpunks/index.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+## Cypherpunk People and Inventions: Tim May
+
+![Tim May](./etc-course-tim-may.png)
+
+Tim May was a founding father of the Cypherpunk movement and dedicated his time to explain the philosophy, explain terms and concepts, organize the mailing groups, and even the physical meetings they held in Silicon Valley.
+
+Mr. May was like a role model who provided counsel, motivation, and direction to the group.
+
+Like many Libertarians, his interests revolved around privacy, reducing arbitrary intervention on the internet, and he had a special disdain for the taxman.
```
Comment (TheCrowbill) [inline content/blog/2022-12-29-ethereum-classic-course-2-the-cypherpunks/index.md:52]:
```suggestion
Like many Libertarians, his interests revolved around privacy, reducing arbitrary intervention on the internet, and he had a special disdain for the tax man.
```

File: content/blog/2022-12-29-ethereum-classic-course-2-the-cypherpunks/index.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+He passed away in December of 2018.
+
+## Cypherpunk People and Inventions: Adam Back
+
+![Adam Back](./etc-course-adam-back.png)
+
+Adam Back is another Cypherpunk who in 1997 was working on a way to avoid spam in a peer-to-peer email system. If the internet were to be decentralized, then tech company email services should be avoided and each user should be able to prevent spam.
+
+His solution was a system called HashCash that used proof of work; the heart of the technology in Bitcoin, and the heart of the consensus mechanism that ETC uses today; as a way of blocking spammers.
```
Comment (TheCrowbill) [inline content/blog/2022-12-29-ethereum-classic-course-2-the-cypherpunks/index.md:62]:
```suggestion
As a way of blocking spammers, he designed a system using proof of work called HashCash. Proof of work is at the heart of the technology used in Bitcoin and is the consensus mechanism that ETC uses today.
```
Edited for clarity, to remove awkward punctuation and repeated wording.

File: content/blog/2022-12-29-ethereum-classic-course-2-the-cypherpunks/index.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+Like HashCash, Bit Gold, and B-money, some of RPOW’s design was incorporated into Bitcoin by Satoshi Nakamoto and also forms part of ETC’s blueprint.
+
+Hal Finney passed away in August of 2014.
+
+## Next Class
+
+![The four technologies that form part of BTC and ETC.](./etc-course-pioneer-tech-btc-etc.png)
+
+In our next class we will explain the four precursor technologies invented by these Cypherpunks that eventually gave way and formed part of Bitcoin and what is now Ethereum Classic:
```
Comment (TheCrowbill) [inline content/blog/2022-12-29-ethereum-classic-course-2-the-cypherpunks/index.md:100]:
```suggestion
In our next class we will explain the four precursor technologies invented by these Cypherpunks that eventually gave way to and formed part of Bitcoin and what is now Ethereum Classic:
```

**Review by TheCrowbill — DISMISSED** [review]

**Review by TheCrowbill — APPROVED** [review]
LGTM

**Review by bobsummerwill — APPROVED** [review]
LGTM.  Thanks for the tweaks.

**Pull request merged**